{{ cookiecutter.package_name }}/setup.cfg

```diff
@@ -36,7 +36,7 @@ select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E901,E902
 exclude = extern,sphinx,*parsetab.py
 
 [metadata]
-package_name = {{ cookiecutter.package_name }}
+package_name = {{ cookiecutter.module_name }}
```

Comment:
Module name is the name of the python module where as package name is the human readable name. I.e. 'astropy' is the module name and 'Astropy' is the package name. At least that's the idea.